Exercise/compute_lowest_terms.py

```diff
@@ -1,40 +1,73 @@
 # TODO: complete the function "lowest_terms" below
 
 def lowest_terms(x):
+	"""
+	Reducing fractions basically requires finding factors and dividing by them.
+	Containers (lists) are created e.g cont, factors and results to hold these
+	generated values.
+	"""
 	x = x.split('/')
-	l = []
-	fac = []
-	res = []
+	cont = []
+	factors = []
+	result = []
+
+	# count identifies the number of negatives in the fraction to determine its sign
 	count = 0
+
+	# Separating negatives from values
 	for i in x:
 		if '-' in i:
 			x[x.index(i)] = i[1:]
 			count += 1
+
+	# Converting strings to integers
 	for j in x:
-		l.append(int(j))
-	copy = l[:]
+		cont.append(int(j))
+
+	# copy is a copy of the prepared input that wouldn't be altered
+	copy = cont[:]
+
+	"""
+	Only the factors of the minimum value of either numerator or denominator are required
+	because any number greater than the minimum value wouldn't be a common factor for both
+	numerator and denominator and thus wouldn't be able to divide both. Thus we loop
+	through the minimum value 
+	"""
```

Comment:
Unnecessarily verbose comment. "looping through the minimum value" which is your last sentence would do just fine.

Note that it is okay to write comments like these to remind yourself why you did certain things for later (possibly on a different branch), however, these should be removed/shortened for your main branch (code that others will read)

Reply:
Noted